env_utils.py

```diff
@@ -0,0 +1,16 @@
+from dotenv import load_dotenv
+from pydantic_settings import BaseSettings
+
+load_dotenv()
+
+
+# Define MongoDB settings class
+class MongoDBSettings(BaseSettings):
+    mongo_client: str
+    mongo_db_name: str
+    mongo_patrons_collection: str
+    mongo_items_collection: str
+    mongo_bills_collection: str
+
+    class Config:
+        env_file = '.env'
```

Comment:
Why do you have random python files in the root directory?
Where should these files be placed?